src/hooks/useConfirmTransaction.ts

```diff
@@ -155,6 +157,7 @@ const useConfirmTransaction = (props: Props): ReturnProps => {
           destChain,
           receivingWallet.address,
           destToken,
+          walletProvider,
```

Comment:
Not the biggest fan of passing the `walletProvider` here, but it's needed in the SDKv2Signer class [here](https://github.com/wormhole-foundation/wormhole-connect/pull/3698/files#diff-f99342fb595ea5a206b2e09ea318ac2a8af6f59d9c8478b4244a26a90112e53aR129) to get the wallet.